lib/Twig/Template.php

```diff
@@ -603,6 +603,8 @@ protected function getAttribute($object, $item, array $arguments = array(), $typ
                 }
             } else {
                 $methods = get_class_methods($object);
+                // sort values to have consistent behavior with lines 611 to 631
+                sort($methods);
```

Comment:
AFAIK, the doc already says that ``get*`` wins over ``is*``. This sorting fixes this behavior because the recent refactoring done by @nicolas-grekas lost the precedence order (making the first found win instead)